client-app/shared/catalog/components/facet-filter.vue

```diff
@@ -109,19 +192,148 @@ const hasFade = computed(
     (searchedValues.value.length > SHOW_MORE_AMOUNT && !isExpanded.value) ||
     (isAnchorAdded.value && !fadeVisibilityAnchorIsVisible.value),
 );
+const selectedFiltersCount = computed(() => facet.value.values.filter((item) => item.selected)?.length);
+const hasSelected = computed(() => selectedFiltersCount.value > 0);
 </script>
 
-<style scoped lang="scss">
-.fade-bottom {
-  --scrollbar-width: 15px;
+<style lang="scss">
+.facet-filter {
+  .vc-checkbox__label {
+    @apply w-full;
+  }
+
+  &__trigger {
+    @apply border-2 border-r-4;
+
+    &--opened {
+      .facet-filter__trigger-arrow {
+        @apply rotate-180;
+      }
+    }
+  }
+
+  &__wrapper {
+    @apply relative z-10;
+  }
+
+  &__trigger-wrapper {
+    @apply flex items-center gap-2;
+  }
+
+  &__trigger-arrow {
+    @apply ml-2 transition-transform;
+  }
+
+  &__item-input {
+    @apply w-full;
+  }
+
+  &__item-inner {
+    @apply flex items-center;
+  }
+
+  &--collapsable {
+    .facet-filter__content {
```

Comment:
No, use scss variable for modifier. Do not duplicate classes.
Example: https://github.com/VirtoCommerce/vc-theme-b2b-vue/blob/dev/client-app/ui-kit/components/organisms/push-message/vc-push-message.vue#L78